modules/wifi-websockets/pages/db-and-grafana.adoc

```diff
@@ -0,0 +1,65 @@
+= TimescaleDB and Grafana
+
+Having an easy way to visualize your data using Websocket application is great getting started experience. The next logical step for most applications is to provide a storage for the data along with the way to create custom dashboards. For this purpose, using https://www.timescale.com/[TimescaleDB] time series database based on Postgres and https://grafana.com/[Grafana] is a popular choice. For this purpose we provided a ready to use Helm chart that you can deploy in any Kubernetes cluster and get your IoT telemetry application ready to use.
+
+== Deploying
+
+Let's deploy this example and configure it to store and visualize our device temperature data. For that we will need Kubernetes cluster and https://helm.sh/[Helm] available on our machine. In this case we will use https://minikube.sigs.k8s.io/[Minikube], but any other will work just fine.
+
+First, you need to start the Minikube
+
+[source]
+----
+minikube start --addons ingress
+----
+
+Considering, we have our Drogue cloud properly configured in the previous section, we need to create an _Access Token_ to connect to the sandbox.
+
+[source]
+--
+drg admin tokens create
+
+A new Access Token was created:
+
+drg_TOKEN_XXX
```

Comment:
Maybe drg_HASH_xxx . Using Token might be confusing it with the whole which is a token as well.